packages/mask/src/plugins/Trader/SNSAdaptor/trader/TraderDialog.tsx

```diff
@@ -183,7 +194,16 @@ export function TraderDialog() {
                 <Web3ContextProvider value={{ pluginID: NetworkPluginID.PLUGIN_EVM, chainId }}>
                     <AllProviderTradeContext.Provider>
                         <Trader
-                            {...traderProps}
+                            defaultInputCoin={
+                                defaultInputCoin
+                                    ? (inputToken as FungibleToken<ChainId, SchemaType.Native | SchemaType.ERC20>)
```

Comment:
```suggestion
                                    ? (inputToken as FungibleToken<ChainId, SchemaType>)
```

Reply:
Type 'FungibleToken<ChainId, SchemaType>' is not assignable to type 'FungibleToken<ChainId, SchemaType.Native | SchemaType.ERC20>'.